corehq/apps/case_importer/static/case_importer/js/excel_fields.js

```diff
@@ -102,6 +100,9 @@ hqDefine('case_importer/js/excel_fields', [
                 });
                 return _.chain(suggestions).sortBy('distance').pluck('field').value();
             });
+            row.isSystemProperty = ko.computed(function () {
+                return row.isCustom() && systemFields.contains(row.customCaseField());
+            });
```

Comment:
With custom's definition essentially being changed to 'not system property', should we rewrite `isSystemProperty` to not reference whether it is custom or not? Seems like a system property is any property whose name is included in the system fields

Reply:
I don't think so. We only want to return true here if the user has checked the "Create new property" checkbox (which sets `isCustom` to true) and the custom field name matches a system property.

If a field is not custom then the property is either not set (will not be imported) or is one of the `show_in_menu` properties, which I believe do not include system properties.